skt/executable.py

```diff
@@ -148,7 +148,7 @@ def cmd_merge(cfg):
 
     try:
         idx = 0
-        for mb in cfg.get('merge_ref'):
+        for mb in cfg.get('merge_ref', []):
```

Comment:
Is adding default for `get()` really needed since the value can only be a list of merge refs or an empty list (which you are setting in the parser)?